src/ServiceControl.Config/UI/InstanceAdd/ServiceControlAddViewModel.cs

```diff
@@ -147,7 +147,8 @@ protected void UpdateErrorRetention(TimeSpan value)
             ErrorRetention = ErrorRetentionUnits == TimeSpanUnits.Days ? value.TotalDays : value.TotalHours;
         }
 
-        public void OnSelectedTransportChanged()
```

Comment:
Otherwise, we would need to add `DoNotNotifyAbout...` attribute to suppress the fody warning